### DISCUSSION TITLE
Configuration Conflicts Between @web3modal/ethers and @web3modal/solana in the Same Project

### DISCUSSION THREAD

**Discussion by andremfaria** [reown-com/appkit#2843] [Q&A] [answered]
Hi, I am currently working on a project that utilizes both `@web3modal/ethers` and `@web3modal/solana`. While both modals generally function correctly on their own, I'm facing a configuration conflict when switching between them. 

**Issue:**
- When a user opens the EVM modal and then closes it, any subsequent attempt to open the Solana modal displays it using the ethers configuration (Light mode).
- Similarly, if the Solana modal is opened first, any attempt to open the EVM modal afterwards displays using the Solana configuration (Dark theme).

It appears that the configuration of the first opened modal persists and affects the other one, regardless of the intended setup.

**Question:**
Is there a way to ensure that each modal retains its own separate configuration? I need each modal to display correctly according to its respective settings, regardless of the order in which they are opened.

**Steps to Reproduce:**
Reproducable code -> https://github.com/andremfaria/web3modal-evm-and-solana-modal
vid -> 

https://github.com/user-attachments/assets/7c036599-1c2a-4561-9102-011c44e0e9e5

1. Open the EVM modal (`@web3modal/ethers`) configured in Light mode.
2. Close the modal.
3. Open the Solana modal (`@web3modal/solana`), expected to be in Dark mode, but it appears in Light mode instead.

**Comment by rtomas** [comment]
hi @andremfaria 
Could you please ping me again next week? I'm sure we can give you a solution to this problem.

**Comment by enesozturk** [comment, marked as answer]
@andremfaria while the two separate modals seems possible to configured on a single dApp, we don't provide such a functionality in AppKit v5. We don't plan to support and confirm it'll work with such a setup. 

🎊  Good news is that we were already working on to bring multi-chain functionality to AppKit for months. And early this week, we announced `@reown/appkit`, check out the docs as see how you can configure both chains in a single instance: https://docs.reown.com/appkit/react/core/multichain

Please take a look and let us know if you have any questions and issues. Best
